config/application.rb

```diff
@@ -26,6 +32,8 @@ class Application < Rails::Application
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
     config.time_zone = 'Eastern Time (US & Canada)'
 
+    config.support_url = 'https://upaya.18f.gov/contact'
```

Comment:
According to the [Rails Configuration Guide](http://guides.rubyonrails.org/configuring.html#custom-configuration), the convention seems to be to prefix custom configuration with an `x` to signify that it's not a configuration related to Rails components. So, it would be `config.x.support_url` and you can call it via `Rails.configuration.x.support_url` vs `Rails.application.config.support_url`

However, if length and/or readability is a concern, perhaps this would be a candidate for Figaro? `Figaro.env.support_url` vs `Rails.configuration.x.support_url`.